src/frontend/src/lib/components/transactions/AllTransactionsList.svelte

```diff
@@ -24,6 +36,21 @@
 	$: sortedTransactions = transactions.sort((a, b) =>
 		sortTransactions({ transactionA: a, transactionB: b })
 	);
+
+	let selectedBtcTransaction: BtcTransactionUi | undefined;
```

Comment:
Since we are already there, would it be worth it creating the Zod types for those transaction and using safe parse?

Reply:
I have a plan to do it for all the transaction types, but the ETH one is based on an external type. So I will create them soon enough with a zod refine